### PR TITLE
CHE-6330: Line should appear in the middle of the editor after setCursorPosition()

### PR DESCRIPTION
**Pull request by vinokurig**
### What does this PR do?
When navigating to line opened in the new editor line appears in the middle of the editor after `setCursorPosition()` method.

### What issues does this PR fix or reference?
#6330

When navigating to line opened in the new editor, line appears in the middle of the editor after `setCursorPosition()` method.

#### Release Notes

N/A

#### Docs PR

N/A

### CODE REVIEW

**Comment by vinokurig** [comment]
ci-test

File: ide/che-core-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/jso/OrionTextViewShowOptionsOverlay.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 public class OrionTextViewShowOptionsOverlay extends JavaScriptObject {
 
+  public static final String CENTER_VIEW_ANCHOR = "center";
```
Comment (tolusha) [inline ide/che-core-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/jso/OrionTextViewShowOptionsOverlay.java]: Introduce Enum with all possible values.

Comment (vinokurig) [reply ide/che-core-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/jso/OrionTextViewShowOptionsOverlay.java]: done

**Review by tolusha — APPROVED** [review]

**Comment by vinokurig** [comment]
ci-test

**Pull request merged**